src/pages/changePasswordPage/ResetPasswordPage.jsx

```diff
@@ -25,137 +29,177 @@ const ResetPasswordPage = () => {
     newPassword: false,
     confirmPassword: false,
   });
+
+  useEffect(() => {
+    const searchParams = new URLSearchParams(location.search);
+    const token = searchParams.get("token");
+    axios.get(`${API_END_POINT}/api/accounts/password/token_verification/?token=${token}`).then((res)=>{
```

Comment:
If there's no token dont perform this request itself add a conditon

Reply:
I have changed